### DISCUSSION TITLE
Down for Maintenance

### DISCUSSION THREAD

**Discussion by darrylmorley** [caprover/caprover#1783] [General]
Hi all,

What's your process for taking a site down for maintenance with CapRover?  Are there any docker images you use, do you use a custom page?

Just looking for ideas/options. Thanks.

**Comment by githubsaturn** [comment]
Down for maintenance pages are outdated concepts. Generally speaking, all maintenance and deployment work can happen with the live website...

Regardless, you can just use a one pager html and deploy it as a Docker image with a webserver (apache/nginx), something like
:
https://codepen.io/calvinbramlett/pen/WOMzqx
